### ISSUE TITLE
use sru do stress testing with jmeter, I find that  with threads of stress testing increases, the speed of sru becomes slower

### ISSUE THREAD

**Issue by GoBelieve007** [asappresearch/sru#159]
I try to replace our LSTM model with SRU.
when I do stress testing with jmeter, when the nums of threads of stress testing over 10,I find the speed of SRU will be slower and slower.
For example, at the beginning of the test, the SRU will spend 0. 1s to predict a logits,however ,at the end of the test,if will spend 1s or 2s to predict a logits . but  this thing would not happen to  LSTM(tf.contrib.rnn.LSTMBlockFusedCell) .
 I guess may be it due to the SRU model can not make full use of cpu resources when it work with other programs。because when i use SRU to predect something along,it will make use of cpu resources,however, i found sometimes,when i use it in some project,it will use much fewer cpu resources than i set in the program(torch.set_num_threads).
So how could i fix this problem? I want to know are there any details I didn’t notice?  thank you for your reply~

The time cost during a stress testing as follow(10 thread):
begin:  sru time:0.06213831901550293
            sru time:0.06213831901550293
            ....
end：sru time:1.7575221061706543
         sru time:1.7065777778625488
